content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst

```diff
@@ -0,0 +1,106 @@
+==========================
+Scrap during manufacturing
+==========================
+
+During the manufacturing process, the need to scrap manufacturing components or finished products
+may arise. This can be necessary if a component or product is damaged, or unusable for any other
+reason.
+
+By default, scrapping a component or finished product removes it from physical inventory and places
+it in a virtual location titled *Virtual Locations/Scrap*. A virtual location is not a physical
+space, but rather a designation in Odoo used to track items that are no longer in physical
+inventory. For more information about virtual locations, see the documentation about the different
+types of :ref:`locations <manufacturing/management/difference-warehouse-location>`.
+
+.. tip::
+   Scrap orders can be viewed by navigating to :menuselection:`Inventory --> Operations --> Scrap`.
+   Each scrap order shows the date and time the order was created, along with the product and
+   quantity that was scrapped.
+
+   To view the total quantity of each item scrapped, navigate to :menuselection:`Inventory -->
+   Configuration --> Locations`. Remove the :guilabel:`Internal` filter from the
+   :guilabel:`Search...` bar to display virtual locations. Finally, select the
+   *Virtual Locations/Scrap* location.
+
+Odoo *Manufacturing* allows for both components and finished products to be scrapped within a
+manufacturing order. The specific type of item that can be scrapped during a manufacturing order
+depends on the stage of the manufacturing process.
+
+Scrap manufacturing components
+==============================
+
+To scrap components during the manufacturing process, begin by navigating to
+:menuselection:`Manufacturing --> Manufacturing Orders`, then select a manufacturing order or click
+:guilabel:`Create` to configure a new one. If a new manufacturing order is created, select a product
+from the :guilabel:`Product` drop-down menu, then click :guilabel:`Confirm`.
+
+Once the manufacturing order has been confirmed, a :guilabel:`Scrap` button appears at the top of
+the page. Click the button and a :guilabel:`Scrap` pop-up window appears.
+
+.. image:: scrap_manufacturing/scrap-button.png
+   :align: center
+   :alt: The scrap button on a manufacturing order.
+
+From the :guilabel:`Product` drop-down menu on the :guilabel:`Scrap` pop-up window, select the
+component that is being scrapped, then enter the quantity in the :guilabel:`Quantity` field.
+Finally, click :guilabel:`Done` to scrap the component.
+
+.. image:: scrap_manufacturing/scrap-pop-up.png
+   :align: center
+   :alt: The Scrap pop-up window.
+
+.. tip::
+   Before clicking :guilabel:`Mark As Done` on a manufacturing order, only the components of the
+   finished product can be scrapped, not the finished product itself. This is because Odoo recognizes
+   that the finished product cannot be scrapped before it has been manufactured.
```

Comment:
```suggestion
   finished product can be scrapped, **not** the finished product itself. This is because Odoo
   recognizes that the finished product cannot be scrapped before it has been manufactured.
```